jenkins-slaves/jenkins-slave-golang/README.md

```diff
@@ -17,11 +17,4 @@ oc process -f ../../.openshift/templates/jenkins-slave-generic-template.yml \
 For all params see the list in the `../../.openshift/templates/jenkins-slave-generic-template.yml` or run `oc process --parameters -f ../../.openshift/templates/jenkins-slave-generic-template.yml`.
 
 ## Jenkins
-Add a new Kubernetes Container template called `jenkins-slave-golang` (if you've build and pushed the container image locally) and specify this as the node when running builds. If you're using the template attached; the `role: jenkins-slave` is attached and Jenkins should automatically discover the slave for you. Further instructions can be found [here](https://docs.openshift.com/container-platform/3.7/using_images/other_images/jenkins.html#using-the-jenkins-kubernetes-plug-in-to-run-jobs). There are path issues with Jenkins permissions and Go when trying to run a build so easiest way to fix this is to setup the GOLANG path to be same as the WORKSPACE
-```
-export GOPATH=${WORKSPACE}
-go get -v -t ./...
-go build -v
-# if there are Ginkgo tests then also run this!
-$GOPATH/bin/ginkgo -r --cover -keepGoing
-```
+Add a new Kubernetes Container template called `jenkins-slave-golang` (if you've build and pushed the container image locally) and specify this as the node when running builds. If you're using the template attached; the `role: jenkins-slave` is attached and Jenkins should automatically discover the slave for you. Further instructions can be found [here](https://docs.openshift.com/container-platform/3.7/using_images/other_images/jenkins.html#using-the-jenkins-kubernetes-plug-in-to-run-jobs).
```

Comment:
Can you add something to the README about go dep having been removed and now using go modules?

Reply:
Sure. Added a short Modules section